### PR TITLE
Fix python indentation in test_core.py. NFC

### PR DESCRIPTION
**Pull request by sbc100**

### CODE REVIEW

File: em-config [hunk 1]
```diff
@@ -28,4 +28,4 @@ if [ -z "$PYTHON" ]; then
   exit 1
 fi
 
-exec "$PYTHON" "$0.py" "$@"
+exec "$PYTHON" -tt "$0.py" "$@"
```
Comment (kripken) [inline em-config:31]: what does `python -tt` do? (`python --help` does not mention it)

Comment (sbc100) [reply em-config:31]: See the PR description

Comment (kripken) [reply em-config:31]: But the PR description mentions `-t` and this is `-tt`? Is it the same?

Comment (kripken) [reply em-config:31]: (and why don't I see either in python --help..? weird)

Comment (sbc100) [reply em-config:31]: Its in the python man page.   `-tt` is the same as `-t -t` and so the final sentence applies:  `Issue an error when the option is given twice`.

Comment (sbc100) [reply em-config:31]: When I run `python --help` I see `-t     : issue warnings about inconsistent tab usage (-tt: issue errors)`

Comment (sbc100) [reply em-config:31]: You don't see this?

Comment (sbc100) [reply em-config:31]: Oh.. its in the `python2 --help` output but not the `python3 --help` for some reason.

Comment (sbc100) [reply em-config:31]: I guess they removed it : https://github.com/python/cpython/commit/e1b5ac640841e4233bcd0204efcd50fe9b50a6d0

Reverting that part.

Comment (kripken) [reply em-config]: Weird they removed it - very confusing...

**Review by kripken — APPROVED** [review]

**Pull request merged**